examples/rotation_bin/main.rs

```diff
@@ -0,0 +1,83 @@
+mod rotation_bin;
+
+use rotation_bin::{Rotation, RotationBin};
+
+use gtk::glib;
+use gtk::glib::clone;
+use gtk::prelude::*;
+
+fn main() {
+    let application = gtk::Application::new(
+        Some("com.github.gtk-rs.examples.rotation_bin"),
+        Default::default(),
+    );
+
+    application.connect_activate(build_ui);
+    application.run();
+}
+
+fn build_ui(application: &gtk::Application) {
+    let window = gtk::ApplicationWindow::new(application);
+
+    let vbox = gtk::Box::new(gtk::Orientation::Vertical, 6);
+    let rotation_bin = RotationBin::new();
+    let img = gtk::Image::builder()
+        .pixel_size(128)
+        .icon_name("audio-x-generic")
+        .build();
+    rotation_bin.set_child(Some(&img));
```

Comment:
Don't you think it would be more interesting to also rotate an interactive widget like a button? :)

Reply:
Nothing but being clickable I guess. Just to show that this doesn't only work for plain pictures but also interactive widgets